### ISSUE TITLE
Chocolatey downloading 2.38.1 instead of 2.39.0

### ISSUE THREAD

**Issue by mattaw** [NickeManarin/ScreenToGif-Chocolatey#18]
I am very sorry to report that for some reason the community package on chocolatey is downloading `2.38.1` and `ffmpeg 6.0` for some reason. The admin powershell log of an install after a complete uninstall:

```
PS C:\Users\matth> choco install screentogif
Chocolatey v2.2.2
Installing the following packages:
screentogif
By installing, you accept licenses for the packages.
Progress: Downloading ffmpeg 6.0... 100%

ffmpeg v6.0.0 [Approved]
ffmpeg package files install completed. Performing other installation steps.
The package ffmpeg wants to run 'chocolateyInstall.ps1'.
Note: If you don't run this script, the installation will fail.
Note: To confirm automatically next time, use '-y' or consider:
choco feature enable -n allowGlobalConfirmation
Do you want to run the script?([Y]es/[A]ll - yes to all/[N]o/[P]rint): Y

Extracting 64-bit C:\ProgramData\chocolatey\lib\ffmpeg\tools\ffmpeg-release-essentials.7z to C:\ProgramData\chocolatey\lib\ffmpeg\tools...
C:\ProgramData\chocolatey\lib\ffmpeg\tools
Removing extracted archive.
Sleeping for 2 seconds to allow anti-viruses to finish scanning...
Renaming ffmpeg directory to common name (Try 1 / 3)
Successfully renamed directory.
 ShimGen has successfully created a shim for ffmpeg.exe
 ShimGen has successfully created a shim for ffplay.exe
 ShimGen has successfully created a shim for ffprobe.exe
 The install of ffmpeg was successful.
  Software installed to 'C:\ProgramData\chocolatey\lib\ffmpeg\tools'
Progress: Downloading screentogif.portable 2.39.0... 100%

screentogif.portable v2.39.0 [Approved]
screentogif.portable package files install completed. Performing other installation steps.
The package screentogif.portable wants to run 'chocolateyInstall.ps1'.
Note: If you don't run this script, the installation will fail.
Note: To confirm automatically next time, use '-y' or consider:
choco feature enable -n allowGlobalConfirmation
Do you want to run the script?([Y]es/[A]ll - yes to all/[N]o/[P]rint): Y

File appears to be downloaded already. Verifying with package checksum to determine if it needs to be redownloaded.
Hashes match.
Hashes match.
Extracting C:\Users\matth\AppData\Local\Temp\chocolatey\screentogif.portable\2.39.0\ScreenToGif.2.38.1.Portable.x64.zip to C:\ProgramData\chocolatey\lib\screentogif.portable\content...
C:\ProgramData\chocolatey\lib\screentogif.portable\content
 ShimGen has successfully created a shim for ScreenToGif.exe
 The install of screentogif.portable was successful.
  Software installed to 'C:\ProgramData\chocolatey\lib\screentogif.portable\content'
Progress: Downloading screentogif 2.39.0... 100%

screentogif v2.39.0 [Approved]
screentogif package files install completed. Performing other installation steps.
 The install of screentogif was successful.
  Software installed to 'C:\ProgramData\chocolatey\lib\screentogif'

Chocolatey installed 3/3 packages.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).
```

**Comment by NickeManarin** [comment]
Sorry, I forgot that I has to update the ps1 files as well.
I will do so in a few hours.

**Comment by NickeManarin** [comment]
Actually, I sent the update now, I hope that it works. :)

**Comment by mattaw** [comment]
Dang, that was quick, I just pushed a PR to fix it too: #19 !

I am waiting for the fix to get released by chocolatey to try again.

Thanks for fixing it, I can close the PR and could probably change the scripts to use a central source of truth (I think?) if you would find it valuable.
